docs/src/examples/rawkv.md

```diff
@@ -48,4 +48,53 @@ public class Main {
     session.close();
   }
 }
-```
\ No newline at end of file
+```
+
+
+## API V2
+With TiKV version >= 6.1.0, we release a new feature called "TiKV API V2" which provides a new raw key-value storage format allowing the coexistence of transaction data and raw data. Please refer to [v6.10 release notes](https://docs.pingcap.com/tidb/stable/release-6.1.0#ease-of-use) for detail.
```

Comment:
transaction data and raw data -> TxnKV and RawKV 

I personally think the latter is more familiar to TiKV developers.

Reply:
Done.